hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java

```diff
@@ -143,32 +146,45 @@ public void setCandidateRoster(@NonNull final Roster roster, @NonNull final Hand
                 context.storeFactory().readableStore(ReadableRosterStore.class).getActiveRoster();
         final var activeRosterHash = RosterUtils.hash(sourceRoster).getBytes();
         final var candidateRosterHash = RosterUtils.hash(roster).getBytes();
+
+        final var aggregationCalculationStart = Instant.now();
         final var tssPrivateShares =
                 getTssPrivateShares(sourceRoster, maxSharesPerNode, tssStore, candidateRosterHash, context);
         final var candidateRosterParticipantDirectory = computeTssParticipantDirectory(roster, maxSharesPerNode, (int)
                 context.networkInfo().selfNodeInfo().nodeId());
+        final var aggregationCalculationEnd = Instant.now();
+        final var durationOfAggregation = Duration.between(aggregationCalculationStart, aggregationCalculationEnd)
```

Comment:
@edward-swirldslabs is this the aggregation you want to see? Only the time spent in `getTssPrivateShares` and `computeTssParticipantDirectory`? Or should we also be including the time it takes to generate or process the messages?

Reply:
Got it, thanks a lot!